Ventivader/Ventivader/Managers/VentilatorParametersManager.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  VentilatorParametersManager.swift
+//  Ventivader
+//
+//  Created by Al on 05/04/20.
+//  Copyright © 2020 calculator. All rights reserved.
+//
+
+import Foundation
+
+class VentilatorParameterModel {
+    let title: String
+    let units: String?
+    let value: String?
+    var leftFooter: String?
+    var rightFooter: String?
+    var headerHighlight: Bool
+
+    init(title: String,
+         units: String? = nil,
+         value: String? = nil,
+         leftFooter: String? = nil,
+         rightFooter: String? =  nil,
+         headerHighlight: Bool = false) {
+        
+        self.title = title
+        self.units = units
+        self.value = value
+        self.leftFooter = leftFooter
+        self.rightFooter = rightFooter
+        self.headerHighlight = headerHighlight
+    }
+}
+
+final class VentilatorParametersManager {
+    static let shared = VentilatorParametersManager()
+
+    var ventilatorParameters: [VentilatorParameterModel] = [
+        VentilatorParameterModel(title: NSLocalizedString("CONTROL MODE", comment: "Control Panel tile title"),
+                                 value: NSLocalizedString("PC-SIMV", comment: "Control Panel tile value"),
```

Comment:
can you  change this to AC (Assist-Control) which is also called as VCV (Volume Controlled Ventilation). This is our default mode we support currently. Later we might be able to support PCV (Pressure Controlled Ventilation).

Reply:
Done